src/main/java/io/codelex/loops/FizzBuzz.java

```diff
@@ -0,0 +1,27 @@
+package io.codelex.Loops;
+
+public class FizzBuzz {
```

Comment:
šēit ar inut vajadzeja 
`Write a console program in a class named FizzBuzz that prompts the user for an integer, then prints all of the numbers from one to that integer, separated by spaces.` :smile: